processors/project-reference-labeler.ts

```diff
@@ -113,7 +113,18 @@ export async function process(
     console.log()
 
     // get issues that have a checkbox in front of it
-    const urls = issue.body?.match(/(?<=-\s*\[.*?\].*?)(https?:\/{2}(?:[/-\w.]|(?:%[\da-fA-F]{2}))+)/g)
+    var urls = issue.body?.match(/(?<=-\s*\[.*?\].*?)(https?:\/{2}(?:[/-\w.]|(?:%[\da-fA-F]{2}))+)/g)
+    
+    // in github local refs are possible in an issue such as #123 wil link to issues 123 in the same repo as the current issue.
+    // get the current issue url and use it for the url to get issues for refs.
+    const localRefs = issue.body?.match(/(?<=-\s*\[.*?\].*?)(?<=#)([0-9]+)/g)
+    const repoIssuesUrl = issue.html_url.substring(0, issue.html_url.lastIndexOf("/"));
+    // add refs + local url ref to urls
+    localRefs.forEach(function (localRef) {
```

Comment:
can localRefs be null?  Have we tested with issues without local refs?

Reply:
tested with out refs and it totally works fine....